### ISSUE TITLE
[TV4Play] --all-episodes broken if more than 12 episodes in a season

### ISSUE THREAD

**Issue by HarryS** [spaam/svtplay-dl#1090]
Hej,

Using svtplay-dl (master, 2.1+53.gd33186e), the `-A`/--all-episodes switch doesn't grab all episodes now on tv4play, if a season contains more than 12 episodes.

Example: `svtplay-dl https://www.tv4play.se/program/sveriges-m%C3%A4sterkock -A`. It will grab just 24 episodes -- as only 12 are shown per each season, even though there are more episodes available.

Reason: TV4 added some pagination to the episode list, where you need to click "Visa fler" to download more episode infos if the season has more than 12 episodes. It makes some GraphQL API request, which svtplay-dl currently isn't coded to do.

**Comment by Sopor** [comment]
I have now downloaded 19 episodes of 24 and i'm still downloading. So i don't understand why you only get 12 episodes?

```
INFO: Episode 19 of 24
INFO: Url: https://www.tv4play.se/program/sveriges-mästerkock/11036394
INFO: Selected to download hls, bitrate: 4110
INFO: Outfile: sveriges-masterkock.s09e09.sveriges.masterkock.del.9-11036394-tv4play.audio.ts
[665/665][================================================================================] ETA: 0:00:00
INFO: Outfile: sveriges-masterkock.s09e09.sveriges.masterkock.del.9-11036394-tv4play.ts
[286/665][==================================..............................................] ETA: 0:01:35
```

**Comment by Sopor** [comment]
Yes, there are 25 episodes from 2018 and 25 from 2019 and it seems to download 24 of 24 episodes only.
